daweb/zaklady-gitu/struktura-adresaru/cvlekce/textovka-vetrelec.md

```diff
@@ -5,8 +5,13 @@ demand: 3
 
 Zahrajte si na příkazové řádce textovou hru Vetřelec.
 
-1. Stáhněte si k sobě na disk soubory a složky s hrou uvnitř zipového souboru [vetrelec-main.zip](https://github.com/Czechitas-podklady-WEB/vetrelec/archive/refs/heads/main.zip).
-1. Zip soubor rozbalte. V něm se ukrývá celá hra. Pozor ale, obsah nijak nezkoumejte.
+1. Stáhněte si k sobě na disk soubor [vetrelec-main.zip](https://github.com/Czechitas-podklady-WEB/vetrelec/archive/refs/heads/main.zip).
+1. Přesuňte soubor do složky s vašimi projekty (nevytvářejte ale samostatný adresář pro tento projekt – adresář se vytvoří v příštím kroku).
```

Comment:
Adresář vs složka: mícháš ty pojmy úmyslně? Pokud ano, za mě OK :)

Reply:
Záměr to nebyl. Při výkladu to používám obojí, ale v textu by to mohlo být jednotné. Opraveno, díky.